coala_langserver/langserver.py

```diff
@@ -116,17 +141,12 @@ def main():
 
     if args.mode == 'stdio':
         log('Reading on stdin, writing on stdout')
-        s = LangServer(conn=ReadWriter(sys.stdin, sys.stdout))
-        s.listen()
+        stdin, stdout = sys.stdin.buffer, sys.stdout.buffer
```

Comment:
you do not need this line. use these values in your invocation of `start_io_lang_server`